.changes/next-release/bugfix-a5c03c3c-3d5c-454b-8d52-1d125ca0a5cc.json

```diff
@@ -0,0 +1,4 @@
+{
+  "type" : "bugfix",
+  "description" : "fix(amazonq): set Q_WORKER_THREADS only when setting is >0"
```

Comment:
should we change this to "Amazon Q chat `@workspace` uses more than 20% cpu?

Reply:
Fixed